### DISCUSSION TITLE
Awesome Move 优质move项目源码清单

### DISCUSSION THREAD

**Discussion by xiegudong45** [NonceGeek/Move-Camp#218] [Materials]
https://github.com/MystenLabs/awesome-move 里面收录了所有用move实现的公链清单、优质move项目的源码等。 感兴趣的可以看看

**Comment by leeduckgo** [comment]
补充：

https://movefuns.atlassian.net/wiki/spaces/MOVEFUNS/pages/2228225/Move

Aptos 开发心得： https://waaagh.xyz/post/3b1de734-2063-4c72-8af4-f662c5d7245b


awesome-move（Move 精选项目）：
- https://github.com/MystenLabs/awesome-move

move-by-example
- https://move-by-example.com/index.html

Starcoin-cookbook：
- https://cookbook.starcoin.org/

Move-book-zh
- https://github.com/move-language/move/tree/main/language/documentation/book/translations/move-book-zh
- https://github.com/move-dao/move-book-zh
- [Move book](https://github.com/move-dao/move-book-zh/blob/main/src/introduction.md)
- [move-parttern](https://www.move-patterns.com/)
- [aptos dev tutorials](https://aptos.dev/tutorials/aptos-quickstarts)

[Youtube Move 前置学习](https://www.youtube.com/watch?v=fOqTGICNN5I)（这个还是要看的）

[NonceGeekDAO](https://fantastic-delivery-faf.notion.site/Edu-NonceGeekDAO-01f0a1453ba440f79d882e6930c3a7e9)
[Web3-dApp-Camp](https://github.com/NonceGeek/Web3-dApp-Camp)

[MoveTool 工具](https://movetool.app)
https://playground.pontem.network/

标准库：
https://github.com/diem/move/tree/main/language/move-stdlib/sources
